libknet/bindings/rust/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "rust-kronosnet"
+version = "0.1.0"
+authors = ["Christine Caulfield <ccaulfie@redhat.com>"]
+edition = "2018"
```

Comment:
is the edition something specific to rust?

Reply:
yes, and 2018 is the right/only on to use at the moment. think of it as "nobody writes pre C99 code if they can avoid it" ;)